spec/fixtures/clock.rb

```diff
@@ -26,4 +26,6 @@ module Clockwork
   every(1.hour, "Run at 10 past the hour", at: "**:10") do
     "Run at 10 past the hour"
   end
+
+  every(1.hour, "JobNamePassedToTheHandler")
```

Comment:
Am I correct in saying that if this job were added, but your change in the `Test` module wasn't, then loading up this clockfile in tests would fail with a `Clockwork::Manager::NoHandlerDefined` exception?

I'm thinking maybe it's worth adding an [acceptance test](https://github.com/kevin-j-m/clockwork-test/blob/ba908d1f65fd28f85e98a5ef92a838e07ca5aeea/spec/acceptance/clock_spec.rb) for this case, so we're explicit about supporting this, ensuring it continues to work, and if it doesn't, getting feedback on the spec that's specifically there to ensure this works.

My concern being if this fixture file removed this event for whatever reason, we would have nothing else in the library to tell us that's problematic. How does that sound?

Reply:
Totally agree !
Let me know if my update commit seems ok.